upbound/configuration-aws-eks
# Update to uptest v1

### Description of your changes

Updates uptest to version 1:
- use shared Makefile
- add project.mk configuration
- update build module
- cleanup examples



Fixes #

I have:


### How has this code been tested

make render.show
make check-examples
